Source/WebCore/platform/graphics/nicosia/NicosiaContentLayer.cpp

```diff
@@ -35,7 +35,7 @@ namespace Nicosia {
 
 Ref<ContentLayer> ContentLayer::create(Client& client)
 {
-    return adoptRef(*new ContentLayer(client, adoptRef(*new WebCore::TextureMapperPlatformLayerProxyGL)));
+    return adoptRef(*new ContentLayer(client, adoptRef(*new WebCore::TextureMapperPlatformLayerProxyGL(WebCore::TextureMapperPlatformLayerProxy::ContentType::Undefined))));
```

Comment:
What is this for? It's the only case where undefined is used (apart from initial value), can we "define" it somehow?